ui/raidboss/data/05-shb/etc/the_diadem.js

```diff
@@ -20,13 +20,15 @@ export default {
       id: 'Diadem Found Gather Point',
       netRegex: NetRegexes.gameLog({ line: 'You sense a grade .* clouded (?:mineral deposit|rocky outcrop|mature tree|lush vegetation patch).*?', capture: false }),
       netRegexDe: NetRegexes.gameLog({ line: 'Du hast einen? verhüllten? (?:Nutzbaum|Vegetation|Steinbruch|Lagerstätte)? der Güte.*?', capture: false }),
+      netRegexFr: NetRegexes.gameLog({ line: ' (?:Nutzbaum|Vegetation|Steinbruch|Lagerstätte)? der Güte.*?', capture: false }),
```

Comment:
@Akurosia, @JLGarber , @panicstevenson.

can you find me the regex fr.  Sorry,this is beyond my competence, thank you for your help.

Reply:
That's fine, can you tell us what the French lines would be in-game? When a timed node starts in the French client, what does it say for that?

Reply:
I went there quickly for you. I don't know if there are others., for the diadem is:

```Vous détectez une carrière de pierre de niveau inconnu à 499 yalms du côté sud
Vous détectez un carré de végétation évanescent de niveau 80 du côté sud-est```

Reply:
Great. So you would want a regex something like `Vous détectez une? (carrière de pierre|de végétation évanescent).*?`. However, it looks like there are two other kinds of harvest nodes, so we would need to find those as well. I'm not sure where in the game data that would be, but I think one of the other people you notified should probably know.

Reply:
yes, but you have to be careful because there are 2 determinants (`un, une`). Thank you for your, I'm waiting to have all the info to add the regex line.

Reply:
Oh, as to that, [`une?` will match both "un" and "une"](https://regex101.com/r/Qcq9Fa/1), the `?` will match exactly 0 or exactly 1 times.

Reply:
I didn't understand you, but `un` is male and `une` is female.

Reply:
Yes, so you want to match "un" and "une" in one regex, then would be `(un|une)`, right?
And there is another way, written as `une?`, the `?` here means the letter before it (which is `e`) can be matched 0 or 1 times.

Reply:
in German and English, directions are not given, so I think that is too much. second point we also have a feminine and masculine. (`évanescent`, `évanescente`).

Reply:
I think it is ok, but maybe you should match up to `évanescente? de niveau 80 du côté.*?` ?
Because we don't know whether it would be affected by other similar system messages?